### ISSUE TITLE
could not compile `time` (lib) due to 1 previous error

### ISSUE THREAD

**Issue by Allenxuxu** [superlistapp/super_native_extensions#381]
When I use Xcode Product Archive, it's failed.
```
could not compile `time` (lib) due to 1 previous error

```

xcode Version 15.3 
Flutter (Channel stable, 3.16.9, on macOS 14.5 23F79 darwin-arm64
Macbook M3
rustc 1.78.0 (9b00956e5 2024-04-29)

**Comment by knopp** [comment]
Which super_native_extensions version is this with?

**Comment by Allenxuxu** [comment]
> Which super_native_extensions version is this with?

0.8.5

**Comment by knopp** [comment]
This should be fixed in 0.8.16.

**Comment by Allenxuxu** [comment]
```
Because super_native_extensions >=0.8.10 depends on web ^0.5.1 and every version of flutter_test from sdk depends on
  web 0.3.0, super_native_extensions >=0.8.10 is incompatible with flutter_test from sdk.
So, because lifenotes depends on both flutter_test from sdk and super_native_extensions ^0.8.16, version solving
  failed.

```  

It seems that an upgrade to the flutter version is necessary in order to use 0.8.16.

**Comment by knopp** [comment]
super_native_extensions always targets latest stable Flutter version.

**Comment by Allenxuxu** [comment]
> This should be fixed in 0.8.16.

After the upgrade, it's ok. 
Thank you!
